backend/package.json

```diff
@@ -38,8 +39,12 @@
 		"dotenv": "^16.0.3",
 		"express": "^4.18.2",
 		"joi": "^17.9.2",
+		"pg": "^8.11.0",
 		"pino": "^8.14.1",
 		"pino-http": "^8.3.3",
+		"sequelize": "^6.32.1",
+		"sequelize-cli": "^6.6.1",
+		"sequelize-typescript": "^2.1.5",
 		"swagger-ui-express": "^4.6.3",
 		"yaml": "^2.3.0"
 	}
```

Comment:
Should `pg`, `sequelize` and `sequelize-typescript` be normal dependencies instead of devDependencies? Since they are required for running the application.